docs/api.md

```diff
@@ -161,6 +161,7 @@ Following is the supported API format for filter transformations:
                     remove_entry_if_equal: removes the entry if the field value equals specified value
                     remove_entry_if_not_equal: removes the entry if the field value does not equal specified value
                     remove_entry_all_satisfied: removes the entry if all of the defined rules are satisfied
+                    keep_entry: keeps the entry if the set of rules are all satisfied
```

Comment:
```suggestion
                    keep_entry_all_satisfied: keeps the entry if the set of rules are all satisfied
```

For consistency

Reply:
I used a simpler name because there's only 1 "keep entry" rule, as opposed to "remove entry" where there are a bunch of different ones which need to be distinguished from each other.